requirements-server.txt

```diff
@@ -0,0 +1,15 @@
+# Dependencies for notebook server and interaction with JupyterHub
+jupyterhub==1.4.0
+jupyterlab==3.0.14
+notebook==6.3.0
+# Dependencies for jupyter widgets
+bqplot==0.12.25
+ipytree==0.1.8
+ipywidgets-extended==1.0.5
+nglview==2.7.7
+widget-periodictable==2.1.5
+# nbserverproxy. ### are we using this?
```

Comment:
nbserverproxy required?

Reply:
Probably not, I don't remember what was the reason for it to be added. Additionally, `nbserverproxy` is an old name. Now the package is called [`jupyter-server-proxy`](https://github.com/jupyterhub/jupyter-server-proxy).

Reply:
I think the intention was to give access to the AiiDA REST API: cd2e1390edf62730f3b5717a9e83121710c50b0a